sdcm/cluster.py

```diff
@@ -2152,14 +2152,11 @@ def _update_db_packages(self, new_scylla_bin, node_list):
 
         def update_scylla_packages(node, queue):
             node.log.info('Updating DB packages')
+            node.remoter.run('mkdir /tmp/scylla')
```

Comment:
better `mkdir -p`

Reply:
better that I wouldn't create it at all :)
I don't know how it worked before...

regarding -p, I think not having /tmp is something we would like to bypass.

Reply:
ok, looked in the code, we are not running `update_db_packages` in "reuse" mode. So it won't fail